app/validators/questionnaire_schema.py

```diff
@@ -396,6 +409,19 @@ def get_all_dynamic_answer_ids(self, block_id):
             for answer in question.get("dynamic_answers", {}).get("answers", [])
         }
 
+    def get_list_name_for_answer_id(self, answer_id: str) -> str | None:
+        """If the answer is repeating, return the name of the list it repeats over, otherwise None"""
```

Comment:
If the answer is repeating or block type is list collector? Or something like `if answer is repeating, dynamic, part of list collector or repeating section`?

Reply:
Sure, changed it to "If the answer is dynamic or in a repeating block or section, return the name of the list it repeats over otherwise None" to match what runner says